inlong-dataproxy/dataproxy-source/src/main/java/org/apache/inlong/dataproxy/metrics/DataProxyMetricItem.java

```diff
@@ -60,6 +59,11 @@ public class DataProxyMetricItem extends MetricItem {
     public static final String M_SINK_DURATION = "sinkDuration";
     public static final String M_NODE_DURATION = "nodeDuration";
     public static final String M_WHOLE_DURATION = "wholeDuration";
+    //
```

Comment:
Do we need a description for this line or remove it?

Reply:
This empty comment line seems to be for easy reading.
In fact, we just need to use blank lines instead of "//" to mark them as empty comments.

Reply:
@vernedeng cc

Reply:
@healchow You are right, I will use blank lines to replace "//"